UKHomeOffice/cerberus-service
# COP-10250: Add tests for Aggregated counts in the task list

## Description
Add tests for checking Enrichment count for different modes of tasks

Scenario 1: Entity in Task details with previous movements
GIVEN an entity has had previous movements
WHEN the entity is displayed in the task details
THEN the number of previous movements/Examinations/Seizures is displayed under the heading

An entity can be an Driver/Lead passenger, Occupant, Vehicle, Trailer, Haulier, Account

## To Test
run the following tests from spec `task-version-details.spec.js`

`Should verify task version details of unaccompanied task on task details page`
`Should verify task version details of tourist task on task details page`
`Should verify task version details of accompanied task with 2 passengers on task details page`

run the following test from `create-tasks.spec.js` to check the counts of a task with Co-travellers
`Should create a task with a payload contains RoRo Tourist from RBT & SBT`

## Developer Checklist

\* Required
